### PR TITLE
tests, integration-tests: implement the cups-control manual test as a spread test

### PR DESCRIPTION
**Pull request by fgimenez**
The test snap requires snapcraft to be built because of the dependency on lpr. It also requires to define a plug on the network interface, cups-control is not enough.

The test itself requires a pdf printer and then checks that a text file can be sent to it (and the corresponding pdf file is generated) using the test snap when the cups-control plug is connected and that it gets an error when the plug is disconnected.

### CODE REVIEW

File: tests/main/interfaces-cups-control/task.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    echo "===================================="
+
+    echo "When the plug is connected"
+    snap connect cups-control-consumer:cups-control ubuntu-core:cups-control
+    snap interfaces | grep -Pzq "$CONNECTED_PATTERN"
+
+    echo "Then the snap command is able to print files"
+    echo "Hello World" > $TEST_FILE
+    cups-control-consumer.lpr $TEST_FILE
+    while ! test -e $HOME/PDF/test_file.pdf; do sleep 1; done
```
Comment (mvo5) [inline tests/main/interfaces-cups-control/task.yaml]: Nice!

**Comment by mvo5** [comment]
Looks good, once tests are green

**Comment by fgimenez** [comment]
@mvo5 thx :) I've pushed changes to use the test snap from the store instead of installing and executing snapcraft during the test, that will prevent the spread error.

retest this please

**Comment by niemeyer** [comment]
Still broken...

**Comment by fgimenez** [comment]
@niemeyer should be fine now, please have another look.

Cheers,

**Comment by mvo5** [comment]
:+1: But needs de-conflicting.

**Comment by fgimenez** [comment]
@mvo5 deconflicted, thx!

**Comment by niemeyer** [comment]
Tests are broken. Most likely requires at least merging from master.

**Pull request merged**